pkgs/development/python-modules/yamlfix/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
```

Comment:
No longer used after switching to the source from GitHub

```suggestion
```

Reply:
(amend to https://github.com/NixOS/nixpkgs/pull/146994/commits/d83369676cfa770a31d4c4930cee9c2604dcee8d)